docs/3_setup_guide/Delft_Fiat/hydromt_fiat_gui.qmd

```diff
@@ -0,0 +1,229 @@
+  
+  1. Go to **File** >> **Select Working Directory**.
+  2. Select an output folder.
+
+Now you are ready to set up the model.
+
+After creating your model you should end up with a folder structure as describted in the section [Data Input - Folder Structure](data_input.qmd).
+
+## Create Model Boundary
+
+There are multiple options on how to create the model boundaries:
+
+  * Draw polygon/ draw box  
+  * SFINCS model domain 
+  * Upload file (polygon vector file e.g. *.gpkg*, *.geojson*)
+
+![Workflow to create model boundaries.](../../_static/figures/Draw_boundaries.svg){#fig-boundaries}
+
+*GUI instructions*
+
+  1. Go to the **Model Boundary | Quickbuild** -tab and select any of the options in the drop-down menu to create the boundary (@fig-boundaries).
+      a) **Draw polygon**: Click on the map and draw a valid polygon. Double click will end the drawing process.
+      b) **Draw box**: Click on the map and draw a box. Double click will end the drawing process.
+      c) **SFINCS model domain**: If you aleady created a SFINCS model you can select the **/gis-folder**. Do <u>not</u> select the region file itself. It will be recognized automatically by the model.
+      d) **Upload file**: You can upload your own boundary in form of a vector file (e.g. *.gpkg*, *.geojson*) with a valid polygon shape. 
+
+  2. As soon you are done click "**Generate boundary**" and your model boundary is established.
+
+For further details please refer to @fig-model_boundaries.
+
+::: {#fig-model_boundaries}
+
+{{< video ../../_static/videos/hydromt_fiat/Generate_Boundaries.mp4 >}}
+
+Create model boundaries using the "Draw Polygon" method.
+
+:::
+
+## Quickbuild
+If you set up a model in the United States and do not want to use your own data you can simply choose to work with the **quickbuild**. 
+
+The **quickbuild** downloads the data from different sources:  
+
+* **Exposure**: National Structure Inventory (NSI)
+* **Vulnerability curves**: FEMA’s Hazus Program
+* **Social vulnerability index**/**equity**: Census api
+* **Roads**: Open Street Map (OSM)
+
+*GUI instructions*
+
+  1. Go to the **Model Boundary | Quickbuild** -tab and select any of the options in the drop-down menu to create the boundary (@fig-quickbuild).
+  2. After you generated the model boundaries you can select the quickbuild option.
+
+![Workflow to use the "Quickbuild"-option.](../../_static/figures/Quickbuild.svg){#fig-quickbuild}
+
+For further details please refer to @fig-nsi_quickbuild.
+
+::: {#fig-nsi_quickbuild}
+
+{{< video ../../_static/videos/hydromt_fiat/Quickbuild.mp4 >}}
+
+Quickbuild option in the United States using NSI exposure, FEMA Hazus Damage Curves and OSM roads.
+
+:::
+
+## Model from scratch: A simple model
+This section explains the creation of a simple model without additional input by the user. This can be set up globally or in the United States, after you created your model boundaries (please refer to the section *Create Model Boundary*).
```

Comment:
Why without additional input by the user? Isn't that the same as the quick build?

Reply:
I see that you refer to an advanced model, but that isn't shown here? I would keep this section completely generic, and go through all the features that a user can choose. I would not distinguish between simple and advanced. Assume that if it isn't quick-build, we have to describe every possible feature. For each feature we should emphasize that it's optional and that they can leave thing at the default if they want.